code/src/components/Form.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+const Form = ({ handleFormSubmit }) => {
+  const [message, setMessage] = useState("");
+  const [alert, setAlert] = useState(false);
+  const onMessageChange = (e) => {
+    const count = message.length;
+    if (count > 140) {
+      setAlert(true);
+    } else {
+      setAlert(false);
+    }
+    setMessage(e.target.value);
```

Comment:
Maybe you are already aware or maybe I misunderstand the purpose, but there is no message alerted when I reach 140 or try to send more than 140 characters.